ui/src/views/DashboardView/DashboardView.tsx

```diff
@@ -86,58 +78,55 @@ const DashboardView = () => {
 
   return (
     <List
+      title={
+        <>
+          Zaaktypen&nbsp;
+          {isLoading && <Outline.ArrowPathIcon spin={true} aria-label={'Bezig met laden'} />}
+        </>
+      }
+      formProps={{
+        autoComplete: 'off',
+        fields: [
+          {
+            disabled: loadingCatalogi,
```

Comment:
Not really a problem of this PR, but if the table has few items, the catalogue dropdown is partially covered
![Screenshot from 2024-04-09 16-24-23](https://github.com/maykinmedia/open-zaaktypebeheer/assets/19154114/83e88591-9964-4241-8166-9318327c9815)